libs/designsystem/src/lib/components/action-sheet/action-sheet.component.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostBinding,
+  HostListener,
+  Input,
+  Output,
+} from '@angular/core';
+
+import { PlatformService } from '../../helpers/platform.service';
+import { OpenState } from '../../models/open-state';
+import { ModalController } from '../modal/services/modal.controller';
+
+import { ActionSheetConfig } from './config/action-sheet-config';
+import { ActionSheetItem } from './config/action-sheet-item';
+
+@Component({
+  selector: 'kirby-action-sheet',
+  templateUrl: './action-sheet.component.html',
+  styleUrls: ['./action-sheet.component.scss'],
+})
+export class ActionSheetComponent {
+  private state = OpenState.closed;
+  attentionLevel: '2' | '3' = '3';
+  static readonly OPEN_DELAY_IN_MS = 100;
+
+  constructor(
+    private platform: PlatformService,
+    private modalController: ModalController,
+    private elementRef: ElementRef<HTMLElement>
+  ) {}
+
+  @Input() cancelButtonText = 'Cancel';
+  @Input() disabled: boolean = false;
+  @Input() header: string;
+  @Input() subheader: string;
+  @Input() items: Array<ActionSheetItem>;
+  @Input() triggerIconName = 'more';
+  @Input() triggerText?: string;
```

Comment:
Remember to add this to actions-sheet API description in showcase

Reply:
Done